src/wfqc/pckg_dev.py

```diff
@@ -0,0 +1,348 @@
+
+
+import pandas as pd
+import json
+import numpy as np
+import json 
+import random
+from sklearn.model_selection import train_test_split
+
+
+def stratified_split(data, test_size=0.2, random_state=42):
+    rating_bins = [0, 1, 2, 3]
+
+    for d in data:
+        if d['ratingAvg'] == 0:
+            d['rating_bin'] = '0'
+        elif d['ratingAvg'] > 0 and d['ratingAvg'] < 1:
+            d['rating_bin'] = '0-1'
+        elif d['ratingAvg'] >= 1 and d['ratingAvg'] < 2:
+            d['rating_bin'] = '1-2'
+        elif d['ratingAvg'] >= 2 and d['ratingAvg'] <= 3:
+            d['rating_bin'] = '2-3'
```

Comment:
If this is to be part of the package, using a method that simply rounds up (or down) the number would be more elegant. As it is now, it is a bit confusing, e.g., `0-1` does not include 0 nor 1, but `2-3` includes both 2 and 3.

Reply:
This file is just for collecting methods I've written to process data etc, and that I dont think  should be in the final package

Reply:
Yes, I think it would be a good idea to separate scripts for wrangling data used to develop (evaluate) the metric from the code to calculate the metric itself. Maybe these scripts (to create datasets for evaluating the metric) could even be placed in a separate repository where the datasets are also kept?

Reply:
We could indeed create a repo with the "APE in the Wild" data and a Python script used for reading the data, if needed.

Regarding my original question, is the script valid as it is, `0-1` does not include `0` nor `1`, but `2-3` includes both `2` and `3`. Could this affect the quality of the evaluation?